axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -184,8 +184,8 @@ def test_all_states_reachable(self):
         owned_states = set(pair[0] for pair in transitions.keys())
 
         un_callable_states = owned_states.difference(called_states)
-        if un_callable_states:
-            raise AssertionError('The following states are un-callable: {}'.format(un_callable_states))
+        error_message = 'The following states are un-reachable: {}'.format(list(un_callable_states))
+        self.assertEqual(error_message, 'The following states are un-reachable: []')
```

Comment:
from @marcharper 

> I vote to just delete the unreachable states, and let's leave that test in for future submissions.

it's supposed to test whether an FSM has states that can't actually be reached, as was found for the two EvolvedFSM classes .  i thought the lines i originally wrote were clear:

```
if un_callable_states:
  raise AssertionError('The following states are un-callable: {}'.format(un_callable_states)
```

but the coverage tool won't allow them, because the "if" clause is never called.  and now that i wrote down my explanation, please wait while i fix that.  just a moment.